sections/sprint-1/pozyx.tex

```diff
@@ -27,14 +27,19 @@ \subsection{Finding the location of anchors}
 The issue with this approach is that the measurements are not perfect, which might cause the circles to not intersect at exactly one point which will make the positional data seem to jitter.
 
 \subsection{Using UWB}
-To find the position of the tags Pozyx makes use of radio waves. 
-Radio waves travel at the speed of light in vacuum, so by dividing the time of travel between anchors with the speed of light, the distance between them can be found. \\
+To find the position of the tags, Pozyx makes use of radio waves. 
+Radio waves travel at the speed of light, so by dividing the time of travel between anchors with the speed of light, the distance between them can be found.\\
 Because the speed of light is so fast the time measure needs to be very accurate to get the correct distance.
-To achieve this the anchors make use of ultra-wide bandwidth (UWB) \cite{pozyx-UWB}.
-The ultra-wideband signals that the Pozyx devices utilize has a bandwidth of 500 MHz.
-This makes the wavelength very short and by detecting the peak of a narrow "pulse", an accurate time can be found.
-High bandwidth means faster data transfer, which most people would prefer, but if everyone were to use the same frequency the signals would interfere with each other, therefore the use of high-frequency signals is tightly regulated.
-Pozyx can use 500 MHz because it transmits at very low power.
+To achieve this the anchors make use of UWB \cite{pozyx-UWB}.
+UWB is a technology for transmitting data over a wide bandwidth usually wider than 500 MHz.
+The Pozyx sensors use exactly 500 MHz which means they just qualify as a UWB system.
+Because the bandwidth is this wide it makes the wavelength very short and by combining multiple sinusoidal signals with slightly different frequencies the Pozyx tags can create a pulse with a peak which is very narrow and measuring a narrow peak results in a more accurate timing allowing the Pozyx to much more precise than other technologies like Bluetooth and WiFi.
```

Comment:
```suggestion
Because the bandwidth is this wide it makes the wavelength very short, and by combining multiple sinusoidal signals with slightly different frequencies the Pozyx tags can create a pulse with a peak which is very narrow.
Measuring a narrow peak results in a more accurate timing, allowing the Pozyx to be much more precise than other technologies like Bluetooth and WiFi.
```